src/contracts/core/AVSDirectory.sol

```diff
@@ -127,6 +127,14 @@ contract AVSDirectory is
         emit AVSMetadataURIUpdated(msg.sender, metadataURI);
     }
 
+    /**
+     * @notice Called by an operator to cancel a salt that has been used to register with an AVS.
+     * @param salt A unique and single use value associated with the approver signature.
+     */
+    function cancelSalt(bytes32 salt) external {
```

Comment:
Please add a require that the salt is not already spent

Reply:
[2f921f0](https://github.com/Layr-Labs/eigenlayer-contracts/pull/434/commits/2f921f0845762c40dc5dbe73efe177a0dea94936)